LICENSE

```diff
@@ -1,5 +1,7 @@
 Attribution 4.0 International
 
+Copyright: 2020, BIDS Contributors
```

Comment:
This looks like slightly more standard notation from having a look at a couple examples (numpy, pandas).

```suggestion
Copyright (c) 2020, BIDS Contributors.
```